extensions/cpp_region/PyRegion.hpp

```diff
@@ -52,9 +54,14 @@ namespace nupic
     PyRegion(const char * module, BundleIO& bundle, Region * region, const char* className="");
     virtual ~PyRegion();
 
+    // Manual serialization methods
     void serialize(BundleIO& bundle);
     void deserialize(BundleIO& bundle);
 
+    // Capnp serialization methods - not yet implemented for PyRegions
```

Comment:
When reading this code, I still wouldn't know my way through this. When/why might I need/want to use the "Manual serialization methods" vs. the "Capnp serialization methods"? Is one set intended to obsolete the other? Where would I learn any of this?